packages/secure-store/secure-unlock/saltStore.ts

```diff
@@ -0,0 +1,35 @@
+import * as Keychain from 'react-native-keychain'
+import { type KeychainOptions, getKeychainItemById, storeKeychainItem } from '../keychain'
+
+const saltStoreBaseOptions: KeychainOptions = {
+  /* Salt can be accessed on this device */
+  accessible: Keychain.ACCESSIBLE.WHEN_UNLOCKED_THIS_DEVICE_ONLY,
+}
+
+const SALT_ID = (version: number) => `PARADYM_WALLET_SALT_${version}`
```

Comment:
Paradym specific again

Reply:
I feel it is ok to use PARADYM_WALLET as a prefix here. As the 'core' I'd see as paradym wallet. I didn't want to go fully generic "WALLET_SALT"